app/ux/WebView.js

```diff
@@ -454,6 +454,29 @@ Ext.define('Rambox.ux.WebView',{
 
 			webview.executeJavaScript(js_inject);
 		});
+		webview.getWebContents().on('before-input-event', (event, input) => {
+			if (input.type !== 'keyDown') { // event used by default
+				return;
+			}
+			// because keyCode property is not passed
+			const keycode = require('keycodes')
```

Comment:
I think this can be moved outside the event, to prevent calling the require every time the user press a key.